### PR TITLE
Add testReviewedOnly test

### LINKED ISSUE #490: Add Unit tests for the `crowdin download sources --reviewed` case
The new functionality was added In #471 PR without Unit tests covering it. It's a `--reviewed` parameter for the `crowdin download sources` command that allows downloading reviewed source files from the Source Text Review WF step in Crowdin Enterprise.

It would be great to add the Unit tests coverage for this case.

API methods that are used in this feature:
- [Build Reviewed Source Files](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.strings.reviewed-builds.post)
- [Check Reviewed Source Files Build Status](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.strings.reviewed-builds.get)
- [Download Reviewed Source Files](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.strings.reviewed-builds.download.download)

Test - [DownloadSourcesActionTest.java](https://github.com/crowdin/crowdin-cli/blob/cli3/src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java)

### PR DESCRIPTION
**Pull request by Genne23v**
I know this test should be updated, but I would like to discuss below. 

- Although there are many lines added from `--reviewed` implementation, many functions in `DownloadSourceAction.java` are private functions. Only `act()` function is public. Please let me know if you have any expectation for private functions. 
- Maybe my lack of understanding. To me, I see a few branches to add tests in `act()` function. I can change `reviewedOnly`, `properties.getPreserveHierarchy()`. I would like to seek some advice how to approach to this test suite. 
- I haven't found a way to test console messages to test below code. `DownloadSourceAction.java` line 85 to 88. But I'm not sure if this is a good test plan. Let me know what you think. 

```java
if (!isOrganization && this.reviewedOnly) {
            out.println(WARNING.withIcon(RESOURCE_BUNDLE.getString("error.only_enterprise")));
            return;
        }
```
- Lastly, is there a way to check test coverage on my local? 

Sorry that it took so long to get here. But I would appreciate your advise.

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
@Genne23v thanks a lot for your contribution!

It looks like the test is not affecting coverage (according to the CodeCov report comment)

For generating coverage reports locally you could use the `./gradlew build jacocoTestReport` command

@yevheniyJ could you please assist with the best way to write tests for this functionality?

**Comment by Genne23v** [comment]
@andrii-bodnar  I updated two tests. Although the coverage increase is small, it certainly improves. Please let me know if you have any suggestions. Thanks

File: src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java [hunk 1]
```diff
@@ -259,6 +266,48 @@ public void testDryRun() {
...
+    }
+
+    @Test
+    public void testReviewedOnly() throws IOException {
+        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+        PrintStream ps = System.out;
+        System.setOut(new PrintStream(outContent));
+
+        PropertiesWithFiles pb = NewPropertiesWithFilesUtilBuilder
+                .minimalBuiltPropertiesBean(
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java]: Please fix the indentations here and below

**Comment by andrii-bodnar** [comment]
Hi @Genne23v, thanks a lot for the update!

Just left a minor comment, please take a look

**Review by andrii-bodnar — APPROVED** [review]

**Comment by andrii-bodnar** [comment]
@Genne23v thanks for the contribution!

**Comment by Genne23v** [comment]
> @Genne23v thanks for the contribution!

It is my pleasure and I learned a lot from this project. Thanks

**Pull request merged**